app/assets/stylesheets/widget-uswds-styles.css

```diff
@@ -0,0 +1,163 @@
+/* This file was generated by the gulp task 'compileWidgetSass'. */
+
+.tp-usa-js-no-click{
+  pointer-events:none;
+  -webkit-user-select:none;
+     -moz-user-select:none;
+          user-select:none;
+}
+.tp-usa-js-no-click .tp-usa-modal,
+.tp-usa-js-no-click .tp-usa-modal *{
+  pointer-events:auto;
+  -webkit-user-select:text;
+     -moz-user-select:text;
+          user-select:text;
+}
+
+[data-open-modal] *{
```

Comment:
@sanason - did this somehow escape the namespace?

Reply:
The prefixer doesn't prefix attributes. Would a prefixed `data` attribute be valid HTML?

This seems to be the only escapee. I'm going to comment it out for the time being and revisit when I switch our code to use the USWDS modal.